docker-compose.prod.yml

```diff
@@ -1,37 +1,44 @@
 version: '3.7'
 
 services:
-  web:
+  web_prod:
     build:
       context: ./app
-      dockerfile: Dockerfile.prod
-    command: gunicorn app.wsgi:application --bind 0.0.0.0:8000
+      dockerfile: Dockerfile
+    command: ["--production"]
     volumes:
-      - static_volume:/home/app/web/app/static
-      - media_volume:/home/app/web/app/media
+      - static_volume_prod:/home/app/web/app/static
+      - media_volume_prod:/home/app/web/app/media
     expose:
       - 8000
     env_file:
-      - ./.env.prod
+      - ./.env
+    environment:
+      DEBUG: 0
+      DJANGO_SETTINGS_MODULE: app.settings.production
     depends_on:
       - db
-  db:
+
+  db_prod:
     image: postgres:12.0-alpine
     volumes:
-      - postgres_data:/var/lib/postgresql/data/
-    env_file:
-      - ./.env.prod
-  nginx:
+      - postgres_data_prod:/var/lib/postgresql/data/
+    environment:
+      POSTGRES_PASSWORD: ${SQL_PASSWORD}
+      POSTGRES_USER: ${SQL_USER}
+      POSTGRES_DB: ${SQL_DATABASE}
+
+  nginx_prod:
     build: ./nginx
     volumes:
-      - static_volume:/home/app/web/app/static
-      - media_volume:/home/app/web/app/media
+      - static_volume_prod:/home/app/web/app/static
+      - media_volume_prod:/home/app/web/app/media
     ports:
-      - 1337:80
+      - 127.0.0.1:1337:80
     depends_on:
       - web
```

Comment:
Either remove if not necessary or change to `web_prod`.

Reply:
Dropped in 330eec0